dev/benchmarks/macrobenchmarks/lib/src/web/bench_text_layout.dart

```diff
@@ -215,6 +215,9 @@ class BenchBuildColorsGrid extends WidgetBuildRecorder {
   BenchBuildColorsGrid.canvasKit()
       : mode = _TestMode.useCanvasKit, super(name: canvasKitBenchmarkName);
 
+  @override
+  bool get isTracingEnabled => false;
```

Comment:
Can you please leave a comment explaining why we don't want tracing for this test?